spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+
+
+RSpec.describe VehicleFactory do
+  describe '#initialize' do
+    it 'instance of factory' do
```

Comment:
You created an `initialize` method in this class that has `created_vehicles` in it. You would need to test that when you call `factory.created_vehicles` it returns something.